producers/src/main/resources/SoapUI/SDT-Endpoint-soapui-project.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like the only changes that were made to the SoapUI project were internal ones related to a later version being used.  Should this have been included in the commit?  A warning is now displayed if you open it in an earlier version of SoapUI saying it might not work correctly.

It might be a good idea to update it now - just want to check that this wasn't an accidental commit.

Reply:
Confirmed change has been reverted.